### PR TITLE
User login and basic nav bar created

### PR DESCRIPTION
**Pull request by tasty-byte**
Added first name and last name to user login.

Added basic nav bar to facilitate sign-up, log-in, log-out.

Also added shared views director containing partial nav bar, partial flashes, logo.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
```
Comment (tasty-byte) [inline app/controllers/application_controller.rb:2]: Added permitted parameters for first name and last name

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :email,
+                required: true,
+                autofocus: true,
+                input_html: { autocomplete: "email" }%>
+    <%= f.input :first_name,
```
Comment (tasty-byte) [inline app/views/devise/registrations/new.html.erb:11]: Added first name field

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                required: true,
+                autofocus: true,
+                input_html: { autocomplete: "given-name" }%>
+    <%= f.input :last_name,
```
Comment (tasty-byte) [inline app/views/devise/registrations/new.html.erb:15]: Added last name field

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -11,8 +11,10 @@
   </head>
 
   <body>
+    <%= render "shared/navbar" %>
```
Comment (tasty-byte) [inline app/views/layouts/application.html.erb:14]: Rendering partial

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -11,8 +11,10 @@
...
     <%= yield %>
       <p class="notice"><%= notice %></p>
       <p class="alert"><%= alert %></p>
+      <%= render "shared/flashes" %>
```
Comment (tasty-byte) [inline app/views/layouts/application.html.erb:18]: Rendering partial

File: app/views/shared/_flashes.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<% if notice %>
```
Comment (tasty-byte) [inline app/views/shared/_flashes.html.erb:1]: Creating partial - flashes

File: app/views/shared/_navbar.html.erb [hunk 7]
```diff
@@ -0,0 +1,38 @@
+<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
```
Comment (tasty-byte) [inline app/views/shared/_navbar.html.erb:1]: Creating partial - boilerplate navbar

**Comment by daphnevisuals** [comment]
LGTM

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (priscillauribe) [reply app/controllers/application_controller.rb:2]: lgtm

**Pull request merged**